pulpcore/cli/deb/remote.py

```diff
@@ -44,6 +44,8 @@ def remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
         "--policy", type=click.Choice(["immediate", "on_demand", "streamed"], case_sensitive=False)
     ),
     click.option("--distributions", required=True),
+    click.option("--components", help=_("Whitespace separated list of components.")),
```

Comment:
We usually use something like this:
```suggestion
    click.option("--component", "components", multiple=True, help=_("Whitespace separated list of components.")),
```

You'd specify all the components separately that way.

Reply:
Latest implementation uses multiple=True.